pkg/virt-controller/watch/vm_test.go

```diff
@@ -818,6 +818,40 @@ var _ = Describe("VirtualMachine", func() {
 			testutils.ExpectEvents(recorder, FailedDeleteVirtualMachineReason)
 		})
 
+		It("should add ready condition True", func() {
+			vm, vmi := DefaultVirtualMachine(true)
+			addVirtualMachine(vm)
+
+			markAsReady(vmi)
+			vmiFeeder.Add(vmi)
+
+			vmInterface.EXPECT().Update(gomock.Any()).Do(func(obj interface{}) {
+				objVM := obj.(*v1.VirtualMachine)
+				cond := getVmCondition(objVM, v1.VirtualMachineReady)
+				Expect(cond).ToNot(BeNil())
+				Expect(cond.Status).To(Equal(k8sv1.ConditionTrue))
+			}).Return(vm, nil)
+
+			controller.Execute()
+		})
+
+		It("should add ready condition False", func() {
```

Comment:
Can those tests become a table? since the flow is similar and they differ only in the value of the condition...

Reply:
Done